helm/solr/templates/_custom_option_helpers.tpl

```diff
@@ -98,6 +98,10 @@ initContainers:
 topologySpreadConstraints:
   {{- toYaml .Values.podOptions.topologySpreadConstraints | nindent 2 }}
 {{ end }}
+{{- if .Values.podOptions.defaultInitContainerResources -}}
+defaultInitContainerResources:
```

Comment:
discussed offline, using default resource to configure both init container resource should be good for now if we need to custom init container resources as both are light weight init containers.